ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCvePage.tsx

```diff
@@ -236,6 +236,29 @@ function ImageCvePage() {
                 </div>
                 <Divider />
                 <div className="pf-u-background-color-100 pf-u-flex-grow-1">
+                    <Split className="pf-u-px-sm pf-u-pb-md pf-u-align-items-baseline">
```

Comment:
This section was pulled out of the error/loading states below so that if an error occurs in one tab we at least have the ability to change to the other tab.